src/poc/miner_poc_statem.erl

```diff
@@ -206,6 +209,9 @@ mining(enter, _State, Data)->
     {keep_state, Data};
 mining(info, Msg, #data{blockchain=undefined}=Data) ->
     handle_event(info, Msg, Data);
+mining(info, {TxnRef, Result}, #data{txn_ref=TxnRef}=Data) ->
+    lager:info("PoC request submission result ~p", [Result]),
+    {keep_state, Data};
```

Comment:
If you got the result for the `txn_ref` why keep the ref in `Data` at this point?

Reply:
Why not?

Reply:
Hmm, I guess its fine since it will get a new ref on new submission anyway